README.md

```diff
@@ -325,6 +327,18 @@ Go section is shown only in directories that contain `Godeps`, or `glide.yaml`,
 | `SPACESHIP_GOLANG_SYMBOL` | `🐹  ` | Character to be shown before Go version |
 | `SPACESHIP_GOLANG_COLOR` | `cyan` | Color of Go section |
 
+### Rust (`rust`)
```

Comment:
There is also an example section at bottom showcasing various available options.